azure-pipelines.yml

```diff
@@ -14,125 +13,51 @@ variables:
-        displayName: 📒 Create dir for manifest Deploy-Site
-
-      - task: ManifestGeneratorTask@0
-        displayName: 📒 Generate Manifest Deploy-Site
-        inputs:
-          BuildDropPath: apps/pr-deploy-site/dist
-          ManifestDirPath: 'apps/pr-deploy-site/sbom'
-          Verbosity: Verbose
-
-      - task: PublishPipelineArtifact@1
-        displayName: 📒 Publish Manifest Deploy-Site
-        inputs:
-          artifactName: SBom-Deploy-Site-$(System.JobAttempt)
-          targetPath: apps/pr-deploy-site/sbom
-
-      - task: AzureUpload@2
-        displayName: Upload PR deploy site
-        inputs:
-          azureSubscription: $(azureSubscription)
-          BlobPrefix: $(deployBasePath)
-          ContainerName: '$web'
-          SourcePath: 'apps/pr-deploy-site/dist'
-          storage: $(azureStorage)
-
-      - task: GithubPRStatus@0
-        displayName: 'Update PR deploy site github status'
-        inputs:
-          githubOwner: microsoft
-          githubRepo: fluentui
-          githubContext: 'Pull request demo site'
-          githubDescription: 'Click "Details" to go to the deployed demo site for this pull request'
-          # This MUST have a trailing slash, or the links to PR deploy site assets won't work
-          githubTargetLink: $(deployUrl)/
-
-      - script: |
-          yarn e2e $(sinceArg)
-        displayName: Cypress E2E tests
-
-      - template: .devops/templates/cleanup.yml
+resources:
+  repositories:
+    - repository: 1esPipelines
+      type: git
+      name: 1ESPipelineTemplates/1ESPipelineTemplates
+      ref: refs/tags/release
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  parameters:
+    pool:
+      name: Azure-Pipelines-1ESPT-ExDShared
+      image: windows-latest
+      os: windows # We need windows because compliance task only run on windows.
+    customBuildTags: null # send empty tags since it is not supported on GH repo's for PR's
+    customLogIssues: null # send empty tags since it is not supported on GH repo's for PR's
+    stages:
+      - stage: main
+        jobs:
+          - job: BuildTestLint
+            pool:
+              name: '1ES-Host-Ubuntu'
+              image: '1ES-PT-Ubuntu-20.04'
+              os: linux
+            workspace:
+              clean: all
+            steps:
+              - template: .devops/templates/build-test-lint.yml@self
+
+          - job: DeployE2E
+            displayName: Deploy and E2E
+            workspace:
+              clean: all
+            pool:
+              name: '1ES-Host-Ubuntu'
+              image: '1ES-PT-Ubuntu-20.04'
+              os: linux
+            templateContext:
+              outputs:
+                - output: pipelineArtifact
+                  targetPath: $(System.DefaultWorkingDirectory)/apps/pr-deploy-site/dist
+                  artifactName: web
+            steps:
+              - template: .devops/templates/deployE2E.yml@self
+
+              - script: |
```

Comment:
https://github.com/microsoft/fluentui/pull/29702/files#r1377253548

Reply:
This will be deleted later today.